modules/use-process-baselines.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operating/evaluate-security-risks.adoc
+:_module-type: CONCEPT
+[id="use-process-baselines_{context}"]
+= Using process baselines
+
+[role="_abstract"]
+In {product-title}, you can minimize your risks by using process baselining.
+It is a proactive approach to keep your infrastructure secure.
+{product-title} first discovers existing processes and creates a baseline.
+Then, it operates in the default deny-all mode and only allows processes listed in the baseline to run.
+
+[discreet]
+== Process baselines
+
+When you install {product-title}, there is no default process baseline.
+As {product-title} discovers deployments, it creates a process baseline for every container type in a deployment.
+Then it adds all discovered processes to their own process baselines.
+
+[discreet]
+== Process baseline states
+
+During the process discovery phase, all baselines are in an unlocked state.
+
+In an *unlocked* state:
+
+* When {product-title} discovers a new process, it adds that process to the process baseline.
+* Processes do not show up as risks and do not trigger any violations.
+
+After an hour from when {product-title} receives the first process indicator from a container in a deployment, it finishes the process discovery phase.
+At this point:
+
+* {product-title} stops adding processes to the process baselines.
+* New processes (which are not in the process baseline) show up as risks but they do not trigger any violations.
```

Comment:
To avoid parentheses and for clarity, I'd suggest something like:
> New processes that are not in the process baseline show up as risks, but they do not trigger any violations.

Reply:
There are actually two different possible meanings here and on line 43, so my example might not work. One of these is true and the other is not, just not sure which is which:
* New processes that are not in the process baseline show up as risks
* New processes, which are not in the process baseline, show up as risks